pkgs/servers/linkedin-matrix/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib, python3, fetchFromGitLab }:
+
+with python3.pkgs;
```

Comment:
Please move this to the array where it is needed.

Reply:
It seemed ugly to do that since I need it in for `buildPythonPackage` and in `nativeBuildInputs` and `propagatedBuildInputs`.

Reply:
Hmmm, I am not sure. Either is fine for me I guess.